ui/raidboss/data/05-shb/alliance/the_puppets_bunker.ts

```diff
@@ -716,15 +770,17 @@ export default {
     },
     {
       id: 'Puppet Compound 2P Four Parts Resolve',
+      type: 'HeadMarker',
       netRegex: NetRegexes.headMarker({ id: ['004F', '0050', '0051', '0052'] }),
       condition: Conditions.targetIsYou(),
       alertText: (_data, matches, output) => {
-        return {
-          '004F': output.jump({ num: 1 }),
-          '0050': output.cleave({ num: 1 }),
-          '0051': output.jump({ num: 2 }),
-          '0052': output.cleave({ num: 2 }),
-        }[matches.id];
+        const fourPartsMap: { [id: string]: string } = {
+          '004F': output.jump!({ num: 1 }),
+          '0050': output.cleave!({ num: 1 }),
+          '0051': output.jump!({ num: 2 }),
+          '0052': output.cleave!({ num: 2 }),
+        };
+        return fourPartsMap[matches.id];
```

Comment:
Technically this is fine since this would return undefined; should this return Outputs.unknown if undefined though?
(My perceived benefit being that if these headMarker values changed then we're more likely to get a report about ??? than nothing firing)